connector/connector.go

```diff
@@ -30,6 +32,8 @@ type Identity struct {
 
 	Groups []string
 
+	CustomClaims map[string]interface{}
```

Comment:
Why is this `map[string]interface{}` whereas Scopes.CustomClaims has `map[string]struct{}`?

Reply:
Scopes.CustomClaims means that scope exists or not